configs/sample_config.ini

```diff
@@ -0,0 +1,91 @@
+
+[DATASET]
+# Training image (.png) directory.
+TRAIN_DATA_DIR = data/charset_1k
+# Test image (.png) directory.
+TEST_DATA_DIR = data/charset_1k_test
+# Path to label file.
+LABEL_FILE = source/charset_1k.txt
+
+[IMAGE]
+# Image height.
+HEIGHT = 20
+# Image width.
+WIDTH = 20
+# Whether original image is in grayscale or RGB format.
+GRAYSCALE_IN = False
+# Whether to return grayscale image as output.
+GRAYSCALE_OUT = False
+
+[TRAIN_TEST_SPEC]
+# Training batch size.
+TRAIN_BATCH_SIZE = 64
+# Test batch size.
+TEST_BATCH_SIZE = 1
+# Higher: runs slower with better shuffling.
+SHUFFLE_BUFFER_SIZE = 200
+# Allows next n elements to be prepared.
+PREFETCH_BUFFER_SIZE = 20
+
+[DATA_AUG]
+# If true, randomly rotate image by a certain degree.
+RANDOM_ROTATE = True
+# Stddev of normal distribution (rotation angle in degree).
+ROTATE_STDDEV = 0.35
+
+# If true, randomly zoom in on the image by certain percent.
+RANDOM_ZOOM = True
+# Zoom in on the image by at most this percent.
+ZOOM_PERCENT = 4.0
+# Stddev of normal distribution (percentage to zoom in on).
+ZOOM_STDDEV = 0.4
+
+# If true, resize image before output.
+RESIZE = True
+# Resize to this height.
+RESIZE_HEIGHT = 32
+# Resize to this width.
+RESIZE_WIDTH = 32
+
+[MODEL]
+# Model name.
+NAME = ResNet50
+# Shape of input tensor.
```

Comment:
This means we are fix to have a 3 layers and only 3 layers neural net here, right? 
Would it be possible for the configuration to handle something like INPUT_DIM = [32, 32, 3] so we can later specify more layers without the need to change the code?

Reply:
Good idea. This is overlooked because ResNet50 models can only have 3-dimensional image as input.